File: Com.Fubei.Api.OpenApi/Biz/AgentApi/Model/Request/AQueryRefundParam.cs
using System;
using System.Collections.Generic;
using System.Text;
using Com.Illuminati.Galileo.Model;
using Newtonsoft.Json;

namespace Com.Fubei.Api.OpenApi.Biz.AgentApi.Model.Request
{
    /// <summary>
    /// 退款查询
    /// </summary>
    public class AQueryRefundParam : BaseEntity
    {
        [JsonProperty("merchant_id")]
        public int? MerchantId { get; set; }

        [JsonProperty("refund_sn")]
        public string RefundSn { get; set; }

        [JsonProperty("merchant_refund_sn")]
        public string MerchantRefundSn { get; set; }
    }
}

File: Com.Fubei.OpenApi.Vb/Biz/AgentApi/Model/Request/AQueryRefundParam.vb
Imports Com.Illuminati.Galileo.Model
Imports Newtonsoft.Json

Namespace Biz.V2.Model.Request

    ' 退款查询
    ' http://docs.51fubei.com/agent-api/payment/orderRefundQuery.html
    Public Class AQueryRefundParam
        Inherits BaseEntity

        <JsonProperty("merchant_id")>
        Public Property MerchantId As Integer?

        <JsonProperty("refund_sn")>
        Public Property RefundSn As String

        <JsonProperty("merchant_refund_sn")>
        Public Property MerchantRefundSn As String
    End Class
End Namespace